pkg/asset/cluster/metadata.go

```diff
@@ -97,7 +97,10 @@ func (m *Metadata) Generate(_ context.Context, parents asset.Parents) (err error
 	case vspheretypes.Name:
 		metadata.ClusterPlatformMetadata.VSphere = vsphere.Metadata(installConfig.Config)
 	case powervstypes.Name:
-		metadata.ClusterPlatformMetadata.PowerVS = powervs.Metadata(installConfig.Config, installConfig.PowerVS)
+		metadata.ClusterPlatformMetadata.PowerVS, err = powervs.Metadata(installConfig.Config, installConfig.PowerVS)
+		if err != nil {
+			return errors.Errorf("failed to initialize PowerVS: %v", err)
```

Comment:
github.com/pkg/errors is deprecated:
```suggestion
			return fmt.Errorf("failed to initialize PowerVS: %w", err)
```